pyproject.toml

```diff
@@ -1,15 +1,58 @@
-[build-system]
-requires = ["setuptools<72.2", "wheel", "numpy>=1.15", "Cython>=0.29"]
+[project]
+name = "pymoo"
+description = "Multi-Objective Optimization in Python"
+authors = [{ name = "Julian Blank", email = "blankjul@outlook.com" }]
+readme = "README.rst"
+license = "Apache-2.0"
+classifiers = [
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.9',
```

Comment:
I don't think we need to drop 3.9 support (wheels still build fine), so I left this in.